core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerClient.java

```diff
@@ -76,7 +78,19 @@
     newDecorator(CircuitBreakerMapping mapping, CircuitBreakerRule rule) {
         requireNonNull(mapping, "mapping");
         requireNonNull(rule, "rule");
-        return delegate -> new CircuitBreakerClient(delegate, mapping, rule);
+        return newDecorator(CircuitBreakerClientHandler.of(mapping), rule);
+    }
+
+    /**
+     * Creates a new decorator with the specified {@link CircuitBreakerClientHandler} and
```

Comment:
How about adding a simple example code to show how to use `CircuitBreakerClientHandler` to this and the client builder?

Reply:
There are multiple places that accept a `CircuitBreakerClientHandler` as a parameter (`CircuitBreakerClient`, `CircuitBreakerRpcClientBuilder`, `CircuitBreakerClientBuilder`) and I didn't want to add example codes for all methods.
Instead, I've added a sample usage to `CircuitBreakerClientHandler`'s javadocs

Reply:
Thanks.